### ISSUE TITLE
No completion for struct-based options

### ISSUE THREAD

**Issue by eladb** [winglang/wing#5532]
### I tried this:

```js
bring http;

test "get" {
  http.get("http://foo.bar", redirect: {
//                                       ^
}
```

### This happened:

https://github.com/winglang/wing/assets/598796/c505fc1d-13e9-4019-b1e9-88a423448783

No completion here

### I expected this:

The IDE to show me the details of `redirect`

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by MarkMcCulloh** [comment]
> **I expected this:** The IDE to show me the details of redirect

There are three things can think of that can be done here:
1. (easy) Add `(kind)` such as `(struct)` or `(enum)` next to the type name in completions to better signal what it is
2. Signature Help (the window above completions that stays open when you're inside a call) should show details at the top about the current struct expansion field rather than just the struct itself
3. The relevant enum should be at the top of the completion item list. Other options will still be shown though.

**Comment by eladb** [comment]
Can we show the list of enum members so I can select?

**Comment by MarkMcCulloh** [comment]
Yeah, that makes sense. That can be an expansion of (3) where instead of showing the enum itself it shows the members at the top
